samples/bluetooth/tmap_central/src/cap_initiator.c

```diff
@@ -35,7 +35,8 @@ static struct bt_bap_ep *unicast_sink_eps[CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SNK_C
 static struct bt_bap_ep *unicast_source_eps[CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SRC_COUNT];
 
 NET_BUF_POOL_FIXED_DEFINE(tx_pool, CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SNK_COUNT,
-			  CONFIG_BT_ISO_TX_MTU + BT_ISO_CHAN_SEND_RESERVE, 8, NULL);
+			 BT_ISO_SDU_BUF_SIZE(CONFIG_BT_ISO_TX_MTU),
+			 CONFIG_BT_CONN_TX_USER_DATA_SIZE, NULL);
```

Comment:
```suggestion
			  BT_ISO_SDU_BUF_SIZE(CONFIG_BT_ISO_TX_MTU),
			  CONFIG_BT_CONN_TX_USER_DATA_SIZE, NULL);
```

Sorry, it's hard to give the right indentation in reviews :(

Reply:
Hi @Thalley, That's okay.  It may be my editor issue. Thank you very much for pointing it out. do you still see issue here?

Reply:
No problem :)